src/System.ServiceModel.Syndication/tests/TestFeeds/rssOptionalElements.xml

```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0"?>
+<rss version="2.0">
+  <channel customAttribute="customAtt">
+    <title>Liftoff News</title>
+    <link>http://contoso.com/</link>
+    <description>Liftoff to Space Exploration.</description>
+    <language>en-us</language>
+    <lastBuildDate>Tue, 10 Jun 2003 09:41:01 Z</lastBuildDate>
+    <docs>http://contoso.com/rss</docs>
+    <generator>Weblog Editor 2.0</generator>
+    <managingEditor>editor@example.com</managingEditor>
+    <category>TestingFeeds</category>
+    <copyright>Contoso Rights.</copyright>
+    <ttl>60</ttl>
+    <skipHours>
+      <hour>1</hour>
+      <hour>2</hour>
+      <hour>3</hour>
+    </skipHours>
+    <skipDays>
+      <day>Saturday</day>
+      <day>Sunday</day>
+    </skipDays>
+    <textInput>
+      <description>Search Online</description>
+      <title>Search</title>
+      <asdasdasd>asdasdasdasdasd</asdasdasd>
+      <link>http://www.contoso.no/search?</link>
+      <name>input Name</name>
+    </textInput>
+    <!--
+    <image>
+      <url>http://2.bp.blogspot.com/-NA5Jb-64eUg/URx8CSdcj_I/AAAAAAAAAUo/eCx0irI0rq0/s1600/bg_Contoso_logo3-20120824073001907469-620x349.jpg</url>
+      <title>The title is not the same to the original one</title>
```

Comment:
This is commented out XML but this is a very real URL to a real image. Do we want to remove this or sanitize the values?

Reply:
There's a bug with the original implementation of reading `<image>`, which read different values and failed the tests. I'm commenting out this section out for now. We will need to revisit this when we add support for `<image>`.